CMakeLists.txt

```diff
@@ -569,6 +579,24 @@ if(ANDROID)
     target_link_libraries(A2XT_Int INTERFACE log)
 endif()
 
+if(VITA)
+    message("Enabling PS Vita Support.")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -ffast-math -mtune=cortex-a9 -mfpu=neon")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -ffast-math -mtune=cortex-a9 -mfpu=neon")
+
+    # Disable annoying warning for Parameter passing.
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-psabi")
+
+    target_compile_definitions(thextech PRIVATE -DVITA -DVITA=1)
+    target_compile_definitions(thextech PRIVATE -DNO_SCREENSHOT)
```

Comment:
Some of these flags have been renamed or haven’t been implemented on main yet. In general I think we are moving towards flags prefixed with “THEXTECH_”.